tests/tests/DeepTest.php

```diff
@@ -32,104 +32,5 @@ public function test_hasone_deep()
 		$this->assertInstanceOf('Model_Test_Post', $author->test_post);
 		$this->assertEquals(3, $author->test_post->id());
 		$this->assertEquals('changed post', $author->test_post->name());
-
-
-	}
-
-	public function test_association_create()
```

Comment:
Is this test moved somewhere? It's quite essential.

Reply:
Oh, I saw it was skipped. I guess we can just enable it and remove anything to do with uploads.

Reply:
Ok, it's returned.